devices.js

```diff
@@ -6920,6 +6920,14 @@ const devices = [
         extend: gledopto.light_onoff_brightness_colortemp_colorxy,
         meta: {disableDefaultResponse: true},
     },
+    {
+        zigbeeModel: ['GL-MC-001P'],
+        model: 'GL-MC-001P',
+        vendor: 'Gledopto',
+        description: 'Zigbee USB mini LED controller RGB + CCT PRO',
+        extend: gledopto.light_onoff_brightness_colortemp_colorxy,
+        meta: {disableDefaultResponse: true},
```

Comment:
Is disableDefaultResponse really needed here? (can you try to remove it and check if everything works as expected)